examples/webpack.config.local.js

```diff
@@ -47,7 +47,8 @@ const LOCAL_DEV_CONFIG = {
 
 function addLocalDevSettings(config) {
   config.resolve = config.resolve || {};
-  Object.assign(config.resolve, {alias: LOCAL_DEV_CONFIG.resolve.alias});
```

Comment:
can someone enlighten me why js thinks the assign target (config.resolve) is null even if I have line 49?

for reference:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign